spel/minimal-linux.json

```diff
@@ -359,6 +359,7 @@
         "spel_customreponame7": "",
         "spel_customreporpm7": "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm",
         "spel_desc_url": "https://github.com/plus3it/spel",
+        "spel_devnode": "/dev/xvda",
```

Comment:
Update the readme with the new variable, please?

Reply:
kul. Hit it tomorrow.

Reply:
Added...